eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -52,28 +52,31 @@ parameters:
 - name: Platforms
   type: object
   default:
-    Linux_NetCore:
+    Linux:
```

Comment:
Do we want to test net5.0 on linux as well or instead of windows?

Reply:
Maybe? I didn't want to grow/change the matrix as part of this PR. Just getting it in-sync.

Reply:
I'll be OK with testing NET5 on more platforms but wonder about agent use.

Reply:
OK lets get it in sync for now and file an issue for adding more. We are working on more agent support but I don't think adding the linux_net5.0 will break the bank.

Reply:
Filed https://github.com/Azure/azure-sdk-for-net/issues/17771